openfasoc/generators/lc-dco/xschem_rundir/untitled-1.sch

```diff
@@ -0,0 +1,78 @@
+v {xschem version=3.0.0 file_version=1.2 }
```

Comment:
this file is untitled, but in general, we should try to add a header to each of these files.
I am not very versed in xschem file formats...